prow/scripts/cluster-integration/kyma-gke-long-lasting.sh

```diff
@@ -197,11 +196,27 @@ function generateAndExportLetsEncryptCert() {
         --server https://acme-v02.api.letsencrypt.org/directory \
         --dns-google-propagation-seconds=600 \
         -d "*.${DOMAIN}"
-
+	
     TLS_CERT=$(base64 -i ./letsencrypt/live/"${DOMAIN}"/fullchain.pem | tr -d '\n')
     export TLS_CERT
     TLS_KEY=$(base64 -i ./letsencrypt/live/"${DOMAIN}"/privkey.pem   | tr -d '\n')
     export TLS_KEY
+	#encrypt the tls cert
```

Comment:
in the current approach you every time request for a certificate, so it won't solve our problem. I was expecting that at the beginning you should check is cert is stored in the bucket, then check if it is still valid. If so, don't generate a new one, use the existing one. Otherwise, generate a new cert, and replace certificate on the bucket.

Reply:
yes, I wanted to test that we could create the key in the bucket prior to checking for it. If you think I should check for it first I will implement that as well.

Reply:
I get the impression that testing for the file should be in golang, unless there is a good way wrap the gsutil for file not found errors.

Reply:
Beginning thought on how to manage the cert. https://github.com/tuxgoose/test-infra/blob/certuploader/development/tools/cmd/certuploader/main.go